openeo-geotrellis/src/main/java/org/openeo/sparklisteners/GetInfoSparkListener.scala

```diff
@@ -29,21 +28,25 @@ class GetInfoSparkListener extends SparkListener {
     println("*****************************************************************")
   }
 
+  override def onApplicationEnd(applicationEnd: SparkListenerApplicationEnd): Unit = {
```

Comment:
I added logging, because CI gave an error in this branch. Only happened once, but I'll keep the log anyway for if it would happen in the future.

Reply:
ok